src/zenml/integrations/slack/__init__.py

```diff
@@ -15,7 +15,6 @@
 
 from typing import List, Type
 
-from zenml.enums import StackComponentType
```

Comment:
Nice these are gone but why were these not flagged as unused imports previously?

Reply:
We ignore checking `init` files. And actually I think I should add these back in because no way to know how / where they're being used. (See my question in #dumb-questions).